### DISCUSSION TITLE
Error when passing data to options using useFetch + typescript Nuxt3

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#20466] [Questions]
I'm trying to pass options to useFetch, but I get an error

<img width="1068" alt="Снимок экрана 2023-04-24 в 13 53 08" src="https://user-images.githubusercontent.com/58338625/233934008-8f219bd4-1cf0-4e89-b090-44db7651fbb7.png">

**Comment by w934423231** [comment]
我也在找这个问题的答案

**Comment by ndragun92** [comment]
Also one more thing data: data is not needed if u want just data as a name if u want to rename u have to use data: namedData
